lib/testing/src/lib/process-services/pages/start-process.page.ts

```diff
@@ -32,6 +32,7 @@ export class StartProcessPage {
     formStartProcessButton = element(by.css('button[data-automation-id="adf-form-start process"]'));
     startProcessButton = element(by.css('button[data-automation-id="btn-start"]'));
     noProcess = element(by.css('.adf-empty-content__title'));
+    startProcessForm = element(by.css('.adf-start-form-container'));
```

Comment:
This element is part of StartFormComponent ADF component, that is used on start-process component. Maybe it would be good to have a different PO for it.

Reply:
I've created a new PO for Start Form component. Thanks